schema/ddl/20240312160156_create_item.sql

```diff
@@ -0,0 +1,24 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE IF NOT EXISTS items (
+    id char(26) NOT NULL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    price INT NOT NULL,
+    category_id char(26) NOT NULL,
+    maker_id char(26) NOT NULL,
+    created_at timestamptz NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at timestamptz NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    deleted_at TIMESTAMP DEFAULT NULL,
```

Comment:
[must] 他も全部だけど，こうかな！
```suggestion
    deleted_at timestamptz DEFAULT NULL,
```